packages/dds/tree/src/feature-libraries/sequence-field/utils.ts

```diff
@@ -459,6 +459,9 @@ export function tryExtendMark<T>(
 				rhs.lastDetachedBy === lhsReattach.lastDetachedBy &&
 				lhsReattach.detachIndex + lhsReattach.count === rhs.detachIndex
 			) {
+				if ("content" in lhsReattach) {
```

Comment:
This approach seems a bit iffy to me. Is there a better way?

Reply:
I would just do `(lhsReattach as Revive).content.push(...rhs.content);` since we check that the lhs and rhs marks are of the same type.